### ISSUE TITLE
proiel-lowfat/xml misses texts

### ISSUE THREAD

**Issue by ErwinKomen** [biblicalhumanities/greek-new-testament#37]
The Proiel-Lowfat/xml misses a few texts and parts of texts:
Completey missing:
1. 22-2peter 
1. 23-1john
1. 24-2jon

Partly missing:
1. 25-3john
1. 26-jude

**Comment by jonathanrobie** [comment]
I think these texts are missing from the PROIEL treebanks.  I reported these issues to PROIEL:

https://github.com/proiel/proiel-treebank/issues?q=is%3Aissue+is%3Aclosed

They added a section that details which parts have not been analyzed and closed the issues.  Here is the section.

https://github.com/proiel/proiel-treebank/commit/6a1e38c7ce3fefd16cdd89e01e11bed9821bf37c

Here is the source I worked from:

https://raw.githubusercontent.com/proiel/proiel-treebank/master/greek-nt.xml

I guess the best thing for me to do is add a reference to the sections that are missing in PROIEL, pointing to their documentation?
